.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+./test.md
```

Comment:
- [ ] Your test.md is not ignored, please fix it because it's part of the requirement :+1: